site/content/en/docs/Installation/_index.md

```diff
@@ -57,13 +57,7 @@ The following table lists recent Agones versions and their corresponding require
 | 1.22           | 1.21                  |
 | 1.21           | 1.21                  |
 
-## Best Practices
+## Best Practices {#separation-of-agones-from-gameserver-nodes}
+{{/* keep installation/#separation-of-agones-from-gameserver-nodes permalink */}}
```

Comment:
This line is being rendered: https://ae79bca-dot-preview-dot-agones-images.appspot.com/site/docs/installation/#separation-of-agones-from-gameserver-nodes

Reply:
Gah, I must not have refreshed the site-server render after adding the comment. Fixed.